### PR TITLE
Improve README with pagination information

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

**Comment by curquiza** [comment]
bors try

File: README.md [hunk 1]
```diff
@@ -60,13 +60,14 @@ const searchClient = instantMeiliSearch(
...
 
-- `paginationTotalHits` (`200` by default): The total (and finite) number of hits you can browse during pagination.<br>
-It means, by default, you can browse `paginationTotalHits / hitsPerPage = 200 / 20 = 10` pages during pagination.<br>
-By default, `hitsPerPage` is set to `20` but it can be changed with [`InsantSearch.configure`](https://www.algolia.com/doc/api-reference/widgets/configure/js/#examples)<br>
-If the pagination widget is not set, this parameter is ignored.
-
 - `placeholderSearch` (`true` by default). Displays documents even when the query is empty.
 
+- `paginationTotalHits` (`200` by default): The total (and finite) number of hits you can browse during pagination when using the [pagination widget](https://www.algolia.com/doc/api-reference/widgets/pagination/js/). If the pagination widget is not set, `paginationTotalHits` is ignored.<br>
+It means, by default, you can browse `paginationTotalHits / hitsPerPage = 200 / 20 = 10` pages during pagination.<br>
```
Comment (bidoubiwa) [inline README.md:66]:
```suggestion
Which means that, with a `paginationTotalHits` default value of 200, and `hitsPerPage` default value of 20, you can browse `paginationTotalHits / hitsPerPage` => `200 / 20 ` => `10` pages during pagination. Each of the 10 pages containing 20 results.<br>
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -60,13 +60,14 @@ const searchClient = instantMeiliSearch(
...
 ```
 
-- `paginationTotalHits` (`200` by default): The total (and finite) number of hits you can browse during pagination.<br>
-It means, by default, you can browse `paginationTotalHits / hitsPerPage = 200 / 20 = 10` pages during pagination.<br>
-By default, `hitsPerPage` is set to `20` but it can be changed with [`InsantSearch.configure`](https://www.algolia.com/doc/api-reference/widgets/configure/js/#examples)<br>
-If the pagination widget is not set, this parameter is ignored.
-
 - `placeholderSearch` (`true` by default). Displays documents even when the query is empty.
 
+- `paginationTotalHits` (`200` by default): The total (and finite) number of hits you can browse during pagination when using the [pagination widget](https://www.algolia.com/doc/api-reference/widgets/pagination/js/). If the pagination widget is not set, `paginationTotalHits` is ignored.<br>
```
Comment (bidoubiwa) [inline README.md:65]: I let you chose the one you prefer:

```suggestion
- `paginationTotalHits` (`200` by default): The total (and finite) number of hits you can browse during pagination when using the [pagination widget](https://www.algolia.com/doc/api-reference/widgets/pagination/js/). If the pagination widget is not used, `paginationTotalHits` is ignored.<br>
```

```suggestion
- `paginationTotalHits` (`200` by default): The total (and finite) number of hits you can browse during pagination when using the [pagination widget](https://www.algolia.com/doc/api-reference/widgets/pagination/js/). If the pagination widget is not added in the options, `paginationTotalHits` is ignored.<br>
```

```suggestion
- `paginationTotalHits` (`200` by default): The total (and finite) number of hits you can browse during pagination when using the [pagination widget](https://www.algolia.com/doc/api-reference/widgets/pagination/js/). If the pagination widget is not added in the configuration, `paginationTotalHits` is ignored.<br>
```

File: README.md (hunk 1 above)
Comment (curquiza) [reply README.md:66]:
```suggestion
Which means that, with a `paginationTotalHits` default value of 200, and `hitsPerPage` default value of 20, you can browse `paginationTotalHits / hitsPerPage` => `200 / 20 = 10` pages during pagination. Each of the 10 pages containing 20 results.<br>
```

**Pull request merged**